Tests/X509Tests/DNSNamesTests.swift

```diff
@@ -232,6 +232,10 @@ final class DNSNamesTests: XCTestCase {
             ("1234567.example.com", "example.com", true),
             ("foo.1234567.example.com", "foo.1234567.example.com", true),
             ("foo.example.123", "foo.example.123", false),
+
+            // Trailing period doesn't always match
+            ("foo.com", "example.bar.", false),
+            ("foo.com", "foo.www.", false),
```

Comment:
Should we separate our custom test cases from the webpki cases to make it easier if we ever want to update the tests if webkpi adds new cases?

Reply:
I don't think it's really necessary, there are already custom cases in there, but they're strictly at the bottom. I also had to tweak a few of the WebPKI test cases because they're testing a slightly different model, so we've already diverged.